### DISCUSSION TITLE
xmlstats.do not formatted

### DISCUSSION THREAD

**Discussion by willhallam-sn** [arnoudkooi/SN-Utils#403] [General]
I notice that with SN-Utils enabled on Firefox, when I view the xmlstats.do page the XML isn't formatted.  Is this a known behavior? Thks!

**Comment by arnoudkooi** [comment]
Confirmed I'll add an exclusion for xmlstats.do in the manifest 
Should be available in a couple of days, in the meantime, you can do a view source as a workaround.
I'll update once available

**Comment by arnoudkooi** [comment]
Version 7.0.3.0 should now be available
